### PR TITLE
Storage: Upload error handling

### PR DESCRIPTION
**Pull request by adela-almasan**
https://user-images.githubusercontent.com/88068998/178074709-e389069c-ffac-46c1-a9a1-6a8af876ccf5.mov

### CODE REVIEW

File: packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import { css, cx } from '@emotion/css';
-import { uniqueId, isString } from 'lodash';
+import { isString, uniqueId } from 'lodash';
 import React, { ReactNode, useCallback, useState } from 'react';
-import { DropEvent, DropzoneOptions, FileRejection, useDropzone, Accept } from 'react-dropzone';
+import { Accept, DropEvent, DropzoneOptions, FileRejection, useDropzone } from 'react-dropzone';
```
Comment (joshhunt) [inline packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx:4]: it's not a massive issue, but is there a reason you changed the order of these imports?

Comment (nmarrs) [reply packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx:4]: I'm curious too, on first glance though it seems like an alphabetization re-order (maybe done by IDE automatically?) Not sure if we have import order rule set on alphabetization or not (locally for me it does not appear to complain if not alphabetically sorted)

**Review by nmarrs — APPROVED** [review]
LGTM! Tested locally and works as expected 💯

File: packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx (hunk 1 above)
Comment (adela-almasan) [reply packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx:4]: I think it was done automatically by my IDE as I did not change the order. I'll check the settings and disable if the case.

**Pull request merged**